db/migrate/20190422200243_add_last_reminded_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastRemindedToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :last_tasks_reminder, :datetime
```

Comment:
We can not be bolting on columns to the user table 

Use a user custom field here and add a unique index on it.